### PR TITLE
Blog post on Astro 1.4.0

### PR DESCRIPTION
**Pull request by matthewp**

### CODE REVIEW

File: src/content/blog/astro-140.mdx [hunk 1]
```diff
@@ -0,0 +1,96 @@
+---
+title: 'Astro 1.4.0 Release'
+description: 'Introducing: Responsive partial hydration • Solid.js support • Lit SSR support • Named slots • Global style support • and more!'
```
Comment (matthewp) [inline src/content/blog/astro-140.mdx:3]: woops, update

File: src/content/blog/astro-140.mdx [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+```
+
+Given that these selectors both have the same specificity, the one lowest in order, the one you defined in your Title.astro component will prevail within that component. Any `h1` defined outside of your component will be `onyx` color.
+
+You get the best of both worlds; styles defined in Astro are prioritized given the same specificity, but you can override them by increasing specificity when needed.
+
+The order of CSS in Astro now is:
+
+1. CSS in your `public/` folder that add as `<link>` tags in the head.
+1. CSS you import in your frontmatter, ordered by the order of your import statements (depth-first).
```
Comment (Jutanium) [inline src/content/blog/astro-140.mdx:85]:
```suggestion
1. CSS imported in your frontmatter, in the order of your import statements (depth-first).
```
What do we mean by depth-first?

File: src/content/blog/astro-140.mdx [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+1. CSS defined in Astro style tags.
+
+We're currently writing up full document to explain how CSS ordering works and how you can control CSS priority via ordering and specificity.
```
Comment (Jutanium) [inline src/content/blog/astro-140.mdx:88]:
```suggestion
We're currently writing up full documentation to explain how CSS ordering works and how you can control CSS priority via ordering and specificity.
```

File: src/content/blog/astro-140.mdx [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+## JSX support in Vue
+
+<BlogContentImage src={vueJsx} alt="Code example that shows a JSX component defined with Vue" />
+
+Using the [Vue integration](https://github.com/withastro/astro/blob/d08093f7d4f9762c0edc71e14c796d7ea8038cda/packages/integrations/vue/README.md) you can now use JSX and TSX. Setting `jsx: true` in the integrations options enables the [@vitejs/plugin-vue-jsx](https://github.com/vitejs/vite/tree/main/packages/plugin-vue-jsx) Vite plugin which does most of the work.
```
Comment (Jutanium) [inline src/content/blog/astro-140.mdx:94]:
```suggestion
You can now use JSX and TSX with the [Vue integration](https://github.com/withastro/astro/blob/d08093f7d4f9762c0edc71e14c796d7ea8038cda/packages/integrations/vue/README.md). Setting `jsx: true` in the integrations options enables the [@vitejs/plugin-vue-jsx](https://github.com/vitejs/vite/tree/main/packages/plugin-vue-jsx) Vite plugin, which does most of the work, and you can use any of its options.
```

**Review by Jutanium — APPROVED** [review]
Left copy/clarity edits!

**Comment by sarah11918** [comment]
LGTM! I agree with all of @Jutanium's suggestions!

**Pull request merged**